languages.toml

```
@@ -387,6 +387,13 @@ comment-token = "#"
 indent = { tab-width = 2, unit = "  " }
 language-server = { command = "cmake-language-server" }
 
+[[language]]
+name = "make"
+scope = "source.make"
+file-types = ["Makefile", "makefile", "justfile", ".justfile"]
```

`"justfile"` will cover files with `justfile` as their name or as their extension.
```suggestion
file-types = ["Makefile", "makefile", "justfile"]
```

---

Does that include `.justfile` by itself?

---

Ah, that is what this is meant to cover, my mistake.

---

I think we should remove just, there is https://github.com/IndianBoy42/tree-sitter-just